skimage/feature/_texture.pyx

```diff
@@ -103,7 +103,7 @@ def _local_binary_pattern(double[:, ::1] image,
     image : (N, M) double array
         Graylevel image.
     P : int
-        Number of circularly symmetric neighbour set points (quantization of
+        Number of circular symmetric neighbour set points (quantization of
```

Comment:
I think this should be reverted. "circularly" may not be in standard dictionaries but a quick Google search suggests plenty of references in the technical literature. Same for other corrections of "circularly symmetric".

Reply:
```suggestion
        Number of circularly symmetric neighbour set points (quantization of
```